shell/common/api/electron_api_native_image_mac.mm

```diff
@@ -4,6 +4,7 @@
 
 #include "shell/common/api/electron_api_native_image.h"
 
+#include <regex>
```

Comment:
This is a banned library in chromium https://chromium-cpp.appspot.com/ , hence the lint error.

Can do instead:

```c++
size_t pos = name.find(to_remove);
if (pos != std::string::npos) {
   name.erase(pos, to_remove.length());
}
```